cmd/object-handlers.go

```diff
@@ -1298,45 +1330,33 @@ func (api objectAPIHandlers) CopyObjectPartHandler(w http.ResponseWriter, r *htt
 		return
 	}
 
+	// Get the object.
+	var startOffset int64
+	length := srcInfo.Size
+	if rs != nil {
+		startOffset, length, _ = rs.GetOffsetLength(srcInfo.Size)
+	}
```

Comment:
The above block can be replaced with just `startOffset, length, _ := rs.GetOffsetLength(srcInfo.Size)` - it will work even if rs is nil.

Reply:
> The above block can be replaced with just `startOffset, length, _ := rs.GetOffsetLength(srcInfo.Size)` - it will work even if rs is nil.

Yes right